inc/Project.php

```diff
@@ -431,4 +440,29 @@ public function get_last_updated_time(): ?string {
 	public function set_last_updated_time( string $time ): bool {
 		return (bool) gp_update_meta( $this->project->id, static::UPDATE_TIME_KEY, $time, 'project' );
 	}
+
+	/**
+	 * Returns the project's version number.
+	 *
+	 * @since 3.0.0
+	 *
+	 * @return null|string Version number if stored, null otherwise.
+	 */
+	public function get_version(): ?string {
+		$time = gp_get_meta( 'project', $this->project->id, static::VERSION_KEY );
+
+		return $time ?: null;
```

Comment:
```suggestion
		return $version ?: null;
```

Reply:
Already changed in https://github.com/wearerequired/traduttore/pull/133/commits/38762869a4c3db29e20c323386f6218a7f8e85c2 🙂